code-challenges.test.js

```diff
@@ -34,13 +100,75 @@ const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
 
 // b) Create the function that makes the test pass.
 
+  // Pseudo coding:
+  // input:arr
+  // output: returns an array of only the REMAINDERS of the numbers when divided by 3.
+  // process: create a function name: reMain3
+  // create a variable: filArr
+  // in filArr use .filter method on arr, declear filter using value, and use typeof(value) method is equal to "number" to get value thats a number.
+  // then return .map method on filArr, declear map using value, and have the value % 3
+  // pass test
+
+  // const reMain3 = (arr) => {
+  //   const filArr = arr.filter((value) => typeof(value) === "number")
+  //   return filArr.map((value) => value % 3)
```

Comment:
Nice logic! To make it even cleaner, you can refactor by chaining .map to the end of the .filter, and returning it all.